src/core/features/course/pages/list-mod-type/list-mod-type.ts

```diff
@@ -34,10 +34,14 @@ import { CoreAnalytics, CoreAnalyticsEventType } from '@services/analytics';
 })
 export class CoreCourseListModTypePage implements OnInit {
 
+    static readonly LOAD_MORE_ACTIVITIES = 20; // How many activities should load each time showMoreActivities is called.
+
     sections: CoreCourseSection[] = [];
     title = '';
     loaded = false;
-    courseId?: number;
+    courseId = 0;
+    canLoadMore = false;
+    showSectionId = 0;
```

Comment:
```suggestion
    showSectionIndex = 0;
```
I think this name is better since this is not the section ID, it's the index in the array of sections.